controllers/envtest/volumegroup_controller_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package envtest
+
+import (
+	"context"
+	"time"
+
+	volumegroupv1 "github.com/IBM/csi-volume-group-operator/api/v1"
+	"github.com/IBM/csi-volume-group-operator/controllers/envtest/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Controller", func() {
+	Describe("Test controllers", func() {
+
+		Context("Test VG controller", func() {
+
+			BeforeEach(func() {
+				err := cleanTestNamespace()
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("Should create all the relevant objects", func(done Done) {
+				By("Creating volumeGroup objects")
+				err := utils.CreateResourceObject(Secret, k8sClient)
+				Expect(err).NotTo(HaveOccurred())
+				err = createVolumeGroupObjects()
+				Expect(err).NotTo(HaveOccurred())
+				time.Sleep(1 * time.Second)
+
+				vgObj := &volumegroupv1.VolumeGroup{}
+
+				By("Validating VolumeGroup object created")
+				err = utils.GetNamespacedResourceObject(VGName, Namespace, vgObj, k8sClient)
+				Expect(err).NotTo(HaveOccurred())
+
+				By("Validating VolumeGroupContent object created")
+				_, err = utils.GetVGCObjectFromVG(VGName, Namespace, vgObj, k8sClient)
+				Expect(err).NotTo(HaveOccurred())
+
+				close(done)
+			}, Timeout.Seconds())
+			It("should add and remove volume objects from volumeGroup objects when created before vg", func(done Done) {
+				By("Creating volume objects before volumeGroup objects")
+				err := utils.CreateResourceObject(Secret, k8sClient)
+				Expect(err).NotTo(HaveOccurred())
+				err = utils.CreateResourceObject(StorageClass, k8sClient)
+				Expect(err).NotTo(HaveOccurred())
+				err = createVolumeObjects()
+				Expect(err).NotTo(HaveOccurred())
+				err = createVolumeGroupObjects()
+				Expect(err).NotTo(HaveOccurred())
```

Comment:
```suggestion

				err = createVolumeObjects()
				Expect(err).NotTo(HaveOccurred())

				err = createVolumeGroupObjects()
				Expect(err).NotTo(HaveOccurred())
```
for visual seperation